modules/users/client/components/ProfileViewBasics.component.js

```diff
@@ -2,9 +2,30 @@ import React from 'react';
 import { NamespacesConsumer } from 'react-i18next';
 import '@/config/lib/i18n';
 import PropTypes from 'prop-types';
+import * as languages from '@/config/languages/languages';
 
-export default function ProfileViewBasics({ profile }) {
-  return (<NamespacesConsumer ns="user-profile">{ t => (<>
+export default function ProfileViewBasics({ profile, hasConnectedAdditionalSocialAccounts, isWarmshowersId, socialAccountLink }) {
+  return (<NamespacesConsumer ns={['user-profile', 'languages']}>{ (t) => (<>
+    {/* show div only if replyrate or replyTime is present */}
+    {(profile.replyRate || profile.replyTime) &&
+    <div className="profile-sidebar-section text-muted">
+      {profile.replyRate &&
+      <span>
+        {t('Reply rate {{replyRate}}.', { replyRate: profile.replyRate })}
+      </span>}
+      {profile.replyTime &&
+      <span>
+        <br/>
+        {t('Replies within {{replyTime, fromNow}}.', { replyTime: profile.replyTime })}
+      </span>}
+    </div>}
+    {/* show div only if birthdate or gender is present */}
+    {(profile.birthdate || profile.gender) &&
+    <div className="profile-sidebar-section" >
+      {profile.birthdate && t('{{birthdate, age}} years', { birthdate: new Date(profile.birthdate) })}
+      {(profile.birthdate && profile.gender) && <span>, </span>}
+      <span className={(!profile.birthdate) ? 'text-capitalize' : null }>{t(profile.gender)}.</span>
```

Comment:
We should probably introduce [`classnames`](https://www.npmjs.com/package/classnames) module here for easier conditional management of class selectors.

Reply:
I added that module.